### ISSUE TITLE
Audit #1 – General Overview when logged in –  Custom tooltips are not dismissable and not persistent

### ISSUE THREAD

**Issue by marcus-herrmann** [owncloud/web#4332]
Connected to Audit 1, see #4300 
1.2 General Overview when logged in, URL: https://ocis-a11y.owncloud.works/#/files/list

# Issue
The content of the custom tooltip itself should be hoverable, keeping itself open, in case the cursor is set to by bigger in the user's OS settings. Currently this is not the case.

# Remediation
- Emulating this behaviour: https://www.w3.org/WAI/WCAG21/working-examples/script-hoverable/
- Using `aria-describedby`, see: https://sarahmhigley.com/writing/tooltips-in-wcag-21/
- Using native `title`, which is not ideal, but will do for WCAG/BITV

**Comment by pascalwengerter** [comment]
Closing this since #5018 and #5055 are merged and we're waiting for feedback (and will open new tickets with the requested changes from the `a11y` audit)
